dracut/30ignition/ignition-files.service

```diff
@@ -34,3 +34,10 @@ After=ignition-disks.service
 Type=oneshot
 EnvironmentFile=/run/ignition.env
 ExecStart=/usr/bin/ignition --root=/sysroot --oem=${OEM_ID} --stage=files --log-to-stdout
+# Hack to just always relabel /var on first boot. This is run by
+# systemd-tmpfiles shortly after mounting things, so there isn't a lot of data
+# yet in it. See https://bugzilla.redhat.com/show_bug.cgi?id=1699107. Ideally,
+# systemd would learn to do this at mount time to avoid race conditions. See:
+# https://github.com/systemd/systemd/pull/11903
+ExecStart=/usr/bin/mkdir -p /run/tmpfiles.d/var-relabel.conf
```

Comment:
Did you mean `ExecStart=/usr/bin/mkdir -p /run/tmpfiles.d/`?

Reply:
Yes, thanks! :)